netweaver/monitoring.sls

```diff
@@ -0,0 +1,33 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+
+{% for node in netweaver.nodes if host == node.host %}
+
+prometheus_sap_host_exporter_pkg:
+  pkg.installed:
+    - name: prometheus-sap_host_exporter
+
+# for each node the instance number change so we need to adapt configuration
+sap_host_exporter_configuration:
+  file.managed:
+    - name: /etc/sap_host_exporter.yaml
```

Comment:
`/etc/sap_host_exporter.yaml`, are you sure?
I would use `/etc/sap_host_expoter/sap_host_exporter.yaml`, if we need to add additional things to the configuration folder.
**Anyway, if you do the same in the `ha_cluster_exporter` this is fine, just to be aligned**

Reply:
in ha_cluster exporter we don't need configuration file. (since we don't need to put some special Number which vary from host to host)

I wouldn't have add it  here either :grin: 


regarding the conf file is correct, we use same as ha_cluster_exporter conf path.